samples/apps/smallbank/app/smallbank.cpp

```diff
@@ -41,6 +44,45 @@ namespace ccfapp
   private:
     SmallBankTables tables;
 
+    bool headers_unmatched(RequestArgs& args)
+    {
+      const auto actual =
+        args.rpc_ctx->get_request_header(http::headers::CONTENT_TYPE)
+          .value_or("");
+      if (expected != actual)
+      {
+        return true;
+      }
+      return false;
+    }
+
+    void set_unmatched_header_status(RequestArgs& args)
+    {
+      args.rpc_ctx->set_response_status(HTTP_STATUS_UNSUPPORTED_MEDIA_TYPE);
+      args.rpc_ctx->set_response_header(
+        http::headers::CONTENT_TYPE, http::headervalues::contenttype::TEXT);
+      args.rpc_ctx->set_response_body(fmt::format(
+        "Expected content-type '{}'. Got '{}'.",
+        expected,
+        args.rpc_ctx->get_request_header(http::headers::CONTENT_TYPE)
+          .value_or("")));
+    }
+
+    void set_error_status(RequestArgs& args, int status, std::string&& message)
+    {
+      args.rpc_ctx->set_response_status(status);
+      args.rpc_ctx->set_response_header(
+        http::headers::CONTENT_TYPE, http::headervalues::contenttype::TEXT);
+      args.rpc_ctx->set_response_body(std::move(message));
+    }
+
+    void set_ok_status(RequestArgs& args)
+    {
+      args.rpc_ctx->set_response_status(HTTP_STATUS_OK);
```

Comment:
Since there's no body here, we should (since #1054) be able to set `HTTP_STATUS_NO_CONTENT` and no content-type header.